glrd/manage.py

```diff
@@ -488,7 +508,11 @@ def create_initial_nightly_releases(stable_releases):
     return release_data
 
 def create_single_release(release_type, args, existing_releases):
-    """Generate a release using the release_type, current timestamp and git info, or using provided arguments."""
+    """Create a single release of the specified type."""
```

Comment:
Not part of this PR but for reference - Refactoring: Is there any reason why we don't have a `Release` class with method `create`.

Reply:
https://github.com/gardenlinux/glrd/issues/4